aws-sns-topicpolicy/src/main/java/software/amazon/sns/topicpolicy/CreateHandler.java

```diff
@@ -33,7 +33,7 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
         else if (CollectionUtils.isNullOrEmpty(resourceModel.getTopics())) {
             return ProgressEvent.failed(resourceModel, callbackContext, null, "Value of property Topics must be of type List of String");
         }
-        else if (CollectionUtils.isNullOrEmpty(resourceModel.getPolicyDocument())) {
```

Comment:
Should we also update here: https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-sns/blob/master/aws-sns-topicpolicy/src/main/java/software/amazon/sns/topicpolicy/DeleteHandler.java#L32 and test